omero/sysadmins/server-performance.txt

```diff
@@ -239,6 +239,6 @@ Useful metrics include:
 .. _Check_MK: https://mathias-kettner.de/check_mk.html
 .. _Metrics: http://metrics.dropwizard.io
 .. _JConsole: http://openjdk.java.net/tools/svc/jconsole/
```

Comment:
@rleigh-codelibre - do you know if openjdk.java.net is moving? It still works at the moment.